reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Draft Orders API
+  x-logo:
+    url: "https://andrewjtech123.github.io/imagehost/shopify_logo_white.png"
+    backgroundColor: "#7ab55c"
+  description: >
+    # Draft Orders API 
+      
+      A [Draft Order](https://help.shopify.com/manual/orders/create-orders) represents the status of an order before the customer has completed the checkout process. Using the Draft Orders API you can create a draft order and transition it to a completed order with great flexibility.  The following flows are supported:
+      
+      * Create a draft order that calculates taxes and totals but accept payment from the customer outside of Shopify.
+      * Create a draft order (with taxes and totals) and email your customer a web checkout link to complete purchase and transition to a completed order.
+      * Use Draft Orders API in conjunction with the [Checkouts API](https://help.shopify.com/api/tutorials/sell-through-the-checkout-api) by created a Checkout object based on a draft order (Sales Channel SDK only).  This enables a highly customizable experience for channels.
```

Comment:
Hmm I have no idea, I'm pretty sure none of them are channels since most are still hand picked right?
I'm ok with in or out. Leaving it in might elicit some interesting discussion. Or some painful one. Whatever Jordan thinks!